modules/installation-network-user-infra.adoc

```diff
@@ -114,24 +114,34 @@ for platform containers and provide telemetry data to Red Hat.
 ====
 endif::[]
 
-.Load balancers
-
-Before you install {product-title}, you must provision two layer-4 load
-balancers. The API requires one load balancer and the default Ingress Controller
-needs the second load balancer to provide ingress to applications.
-
+[id="installation-network-user-infra-load-balancers_{context}"]
+=== Load balancers
+
+Before you install {product-title}, you must provision two load balancers that meet the following requirements:
+
+. *API Load Balancer*: Provides a common endpoint for users, both human and machine, to interact with and configure the platform. Configure one of the following strategies:
```

Comment:
@codyhoag the phrase `Configure one of the following strategies` suggests that the below bullets represent two mutually exclusive approaches. They don't. I think that what we are actually trying to state is that

1. In ALL cases, you need a Layer 4 (as opposed to Layer 7) load balancer
2. The proper Load Balancing algorithm to use will vary based on the type of load balancer used, whether it is on prem (e.g. F5, Netscaler) or cloud-based, however, session persistence is not required for the API load balancer to function properly, as there is no session persistence at play. (its important to note this because this may be different for the ingress LB, where it may be serving requests for back end applications that use session persistence).

Reply:
@etsauer I've adjusted that wording in c0649a6302f3d1294b0dc06b1bb3d04aaa49d3d7. Can you confirm?